sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDBFileManager.scala

```diff
@@ -342,6 +363,8 @@ class RocksDBFileManager(
     } else {
       // Delete all non-immutable files in local dir, and unzip new ones from DFS commit file
       listRocksDBFiles(localDir)._2.foreach(_.delete())
+      // TODO: We are using fs here to read the file, checksum verification wouldn't happen
```

Comment:
Do we need a separate SPARK ticket here ?

Reply:
done: https://issues.apache.org/jira/browse/SPARK-51988